optimizer_wrapper.rb

```diff
@@ -890,14 +890,14 @@ def self.apply_zones(vrp)
   end
 
   def self.clique_cluster(vrp, cluster_threshold, force_cluster)
-    if vrp.matrices.size.positive? && vrp.shipments.size.zero? && (cluster_threshold.to_f.positive? || force_cluster) && !vrp.scheduling?
+    if vrp.matrices.any? && vrp.shipments.empty? && (cluster_threshold || force_cluster) && !vrp.scheduling?
```

Comment:
I know it is not really in the socpe of this dev but now that 0 will call clique_cluster we need to fix the condition (vis-a-vis shipmets) because up to today mt-web was protected from clique_cluster (with or without shipment) because it always sends `0` but now `0` will call clique_cluster so we risk calling clique_cluster with shipments if we don't fix it in this PR because mt-web always sends shipments as relations. 

we need to add `vrp.relations.none?{ |r| r.type == :shipment }` (or something similar) to this condition